src/becca/similarity.js

```diff
@@ -35,7 +35,7 @@ const IGNORED_ATTR_NAMES = [
 function filterUrlValue(value) {
     return value
         .replace(/https?:\/\//ig, "")
-        .replace(/www.js\./ig, "")
```

Comment:
Can you explain this change? I haven't been able to pinpoint a reason for it...

Reply:
@j-tirelli , very good catch. I think the reason is an accidental replace when solving the imports.